etc/ci/performance/runner.py

```diff
@@ -281,10 +281,12 @@ def save_result_csv(results, filename, manifest, expected_runs, base):
         'unloadEventStart',
     ]
 
+    successes = list(filter(lambda x: x['domComplete'] != -1, results))
```

Comment:
List comprehensions are more idiomatic in Python:
```
successes = [r for r in results if r['domComplete'] != -1]
```

Reply:
Fixed. There's quite a few more of those :/ Time to file an E-Easy issue?